graph_annotation_test.go

```diff
@@ -73,6 +78,30 @@ func TestCreateGraphAnnotation(t *testing.T) {
 	if err != nil {
 		t.Error("err shoud be nil but: ", err)
 	}
+
+	if annotation.Service != "My-Blog" {
+		t.Error("request sends json including Service but: ", annotation.Service)
+	}
```

Comment:
[nit]
I think the error message is not helpful. ( what does "but" means???)

Something like following code might be better.

```go
	if want := "My-Blog"; annotation.Service != want {
		t.Errorf("request json: Service=%v, want %v", annotation.Service, want)
	}
```
https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures

But, this is nit comment. you can ignore this.